pkg/cstor/pool/v1alpha2/pool_utils.go

```diff
@@ -0,0 +1,98 @@
+package v1alpha2
+
+import (
+	api "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha2"
+	zpool "github.com/openebs/maya/pkg/apis/openebs.io/zpool/v1alpha1"
+	blockdevice "github.com/openebs/maya/pkg/blockdevice/v1alpha2"
+	env "github.com/openebs/maya/pkg/env/v1alpha1"
+	zfs "github.com/openebs/maya/pkg/zfs/cmd/v1alpha1"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func getPathForCSPBdevList(bdevs []api.CStorPoolClusterBlockDevice) ([]string, error) {
+	var vdev []string
+	var err error
+
+	for _, b := range bdevs {
+		path, er := getPathForBDev(b.BlockDeviceName)
+		if er != nil {
+			err = ErrorWrapf(err, "Failed to fetch path for bdev {%s} {%s}", b.BlockDeviceName, er.Error())
+			continue
+		}
+		vdev = append(vdev, path)
+	}
+	return vdev, err
+}
+
+func getPathForBDev(bdev string) (string, error) {
+	bd, err := blockdevice.NewKubeClient().
+		WithNamespace(env.Get("NAMESPACE")).
+		Get(bdev, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+	return bd.Spec.Path, nil
```

Comment:
this is not the correct path. We need to prioritize for something like devLinks[0].Links[0].. please check existing code..
if that doesn't exists, then, for bd.Spec.Path I guess

Reply:
cc: @akhilerm can you review this piece of code?

Reply:
@mynktl . You can refer this [code](https://github.com/openebs/maya/blob/994d3012f3f6ef90fe3368b8a143dbccd43c34f0/cmd/maya-apiserver/cstor-operator/spc/storagepool_create.go#L215) in maya repo, to see how we can access the disk.